snippets/pipeLog.md

```diff
@@ -0,0 +1,13 @@
+### pipeLog
+
+Use `console.log` in a pipeline as this function simply encloses it and returns the passed value. This is especially useful for debugging when you want to log a variable's value before its usage.
+
+Logs a value and returns it.
```

Comment:
This should be the first part of the description of the snippet, detailing the functionality.